_redirects

```diff
@@ -1,2 +1,4 @@
 /v2/webhooks /v2/webhooks/webhooks
 /v2/webhooks/index.html /v2/webhooks/webhooks/
+/v2/domains/collaborators /v2/domains/
+/v2/domains/collaborators/index.html /v2/domains/
```

Comment:
Had no idea we were doing this 😮

Reply:
I started off just leaving a small hint of deprecation on the collaborators page. Then, I said this is silly and found this redirects file!